stock_dashboard_api/models/stock_model.py

```diff
@@ -106,6 +110,23 @@ def get_by_id(cls, pk: int) -> object:  # pylint: disable=C0103
             except (psycopg2.DataError, psycopg2.ProgrammingError, TypeError):
                 return None
 
+    def get_data_for_time_period(self, datetime_from: datetime, datetime_to: datetime) -> list:
+        stock_data_for_time_period = []
+        datetime_from, datetime_to = datetime_from.strftime(DATETIME_PATTERN), datetime_to.strftime(DATETIME_PATTERN)
+        with pool_manager() as conn:
+            query = "SELECT price, created_at FROM stocks_data " \
+                    "WHERE stock_id = %(stock_id)s " \
+                    "AND %(datetime_from)s <= created_at AND created_at <= %(datetime_to)s " \
+                    "ORDER BY created_at;"
+            try:
+                conn.cursor.execute(query, {'stock_id': self.pk,
+                                            'datetime_from': datetime_from,
+                                            'datetime_to': datetime_to})
+                stock_data_for_time_period = conn.cursor.fetchall()
+            except (psycopg2.DataError, psycopg2.ProgrammingError, TypeError):
+                return stock_data_for_time_period
```

Comment:
What is the reason of this return?
So, in case we've got a exception, we still just return the data? (but empty list)